esi/include.go

```diff
@@ -36,6 +37,11 @@ func (i *includeTag) loadAttributes(b []byte) error {
 	return nil
 }
 
+func sanitizeURL(u string, reqUrl *url.URL) string {
+	parsed, _ := url.Parse(u)
+	return reqUrl.ResolveReference(parsed).String()
```

Comment:
@darkweak this doesn't seem to work as I'd expect.

I'm dumping this:
```go
	println(req.Host)
	println(rq.URL.Host)
```

just after having created the child request and getting

```
localhost:9080
domain.com:9080
```

it seems `localhost` gets lost and the new code doesn't work at all, it's trying to connect to `domain.com` which I'm not exactly sure where it comes from.

Reply:
NVM it's actually hardcoded in the fixtures, my bad.